tests/connect/codec.go

```diff
@@ -0,0 +1,29 @@
+package slinky_test
+
+import (
+	enccodec "github.com/cosmos/evm/encoding/codec"
+
+	amino "github.com/cosmos/cosmos-sdk/codec"
+	"github.com/cosmos/cosmos-sdk/codec/testutil"
+	"github.com/cosmos/cosmos-sdk/types/module"
+	sdktestutil "github.com/cosmos/cosmos-sdk/types/module/testutil"
+	"github.com/cosmos/cosmos-sdk/x/auth/tx"
+)
+
+// MakeConfig creates a new EncodingConfig and returns it
+func MakeTestEncodingConfig(modules ...module.AppModuleBasic) sdktestutil.TestEncodingConfig {
+	cdc := amino.NewLegacyAmino()
+	interfaceRegistry := testutil.CodecOptions{}.NewInterfaceRegistry()
+	mb := module.NewBasicManager(modules...)
+	mb.RegisterInterfaces(interfaceRegistry)
+	codec := amino.NewProtoCodec(interfaceRegistry)
+	enccodec.RegisterLegacyAminoCodec(cdc)
+	enccodec.RegisterInterfaces(interfaceRegistry)
```

Comment:
we need register [here](https://github.com/strangelove-ventures/interchaintest/blob/main/dockerutil/keyring.go#L59)